corehq/apps/users/static/users/js/edit_commcare_user.js

```diff
@@ -1,15 +1,40 @@
-/* globals hqDefine */
-hqDefine('users/js/edit_commcare_user', function () {
-    var initialPageData = hqImport('hqwebapp/js/initial_page_data'),
-        couchUserId = initialPageData.get('couch_user_id');
+hqDefine('users/js/edit_commcare_user', [
+    'jquery',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/alert_user',
+    'analytix/js/google',
+    'hqwebapp/js/multiselect_utils',
+    'jquery-textchange/jquery.textchange',
+    'hqwebapp/js/widgets',
+    'registration/js/password',
+    'nic_compliance/js/encoder',
+    'select2/dist/js/select2.full.min',
+    'hqwebapp/js/ui_elements/ui-element-langcode-button',
+    'hqwebapp/js/ui_elements/ui-element-input',
+    'hqwebapp/js/ui_elements/ui-element-checkbox',
+    'hqwebapp/js/ui_elements/ui-element-input-map',
+    'hqwebapp/js/ui_elements/ui-element-key-val-list',
+    'hqwebapp/js/ui_elements/ui-element-key-val-mapping',
+    'hqwebapp/js/ui_elements/ui-element-select',
+    'hqwebapp/js/ui-element', // todo cleanup ui-element imports
```

Comment:
The ui-element imports aren't yet requirejs-compliant, and they're likely to be annoying because `ui-element-key-val-mapping` supports multimedia upload, which isn't yet supported by requirejs. Do you know if these are actually used on this page? I'm wondering if maybe the group membership area used to use them before we had multiselect utils...then they could just be deleted.

Reply:
I don't think they are actually being used. I tested it without this and it appeared to work fine...but maybe I'm missing something from a feature flag, etc

Reply:
Given that this is going through QA and hopefully that'll turn up any problems, could you just delete them?

Reply:
yep!